src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -179,6 +180,10 @@ export default function ConfirmedAppointmentDetailsPage() {
     videoKind !== VIDEO_TYPES.clinic &&
     videoKind !== VIDEO_TYPES.gfe;
 
+  const isCovidVaccine = appointment.vaos.isCOVIDVaccine;
+  const hasCovidPhone = isValidCovidPhoneNumber(facility);
```

Comment:
We could probably do this isValidCovidPhoneNumber check inside FacilityAddress, and fall back to the main phone in there.